### PR TITLE
write delete forwarding rule api docs

### PR DESCRIPTION
**Pull request by xrrlzzq**
The API Docs for deleting forwarding rule

### CODE REVIEW

File: source/includes/gcp/_forwarding_rules.md [hunk 1]
```diff
@@ -1 +1,15 @@
...
+
+```shell
+curl -X DELETE \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/gcp/test-area/forwardingrules/5459493658413662148"
+```
+
+<code>DELETE /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/forwardingrules/:id</code>
+
+Destroy an existing global forwaring rule.
```
Comment (SeventhOfNine) [inline source/includes/gcp/_forwarding_rules.md]: typo:  `forwarding`

Comment (evanluc) [inline source/includes/gcp/_forwarding_rules.md]:
```suggestion
Delete an existing global forwarding rule.
```

**Review by evanluc — COMMENTED** [review]
Other than my tiny suggestion, looks good to me!

File: source/includes/gcp/_forwarding_rules.md (hunk 1 above)
Comment (simongodard) [reply source/includes/gcp/_forwarding_rules.md]: Shouldn't we use the word: `Delete` instead of `Destroy`?

Comment (xrrlzzq) [reply source/includes/gcp/_forwarding_rules.md]: I have checked other delete docs, it always use destroy, so i am not very sure which word should i use

Comment (evanluc) [reply source/includes/gcp/_forwarding_rules.md]: @xrrlzzq if everywhere else uses destroy stick to it :)

**Review by evanluc — APPROVED** [review]
You can switch back to destroy if it's more consistent with other docs as you said above.

File: source/includes/gcp/_forwarding_rules.md (hunk 1 above)
Comment (xrrlzzq) [reply source/includes/gcp/_forwarding_rules.md]: Ok,i will changed back and then merge, Thanks

**Pull request merged**